src/p11_child/p11_child_nss.c

```diff
@@ -100,6 +100,91 @@ static char *get_key_id_str(PK11SlotInfo *slot, CERTCertificate *cert)
     return key_id_str;
 }
 
+int init_verification(struct cert_verify_opts *cert_verify_opts,
```

Comment:
The functions could be made static to make it even more clear that they only belong to this file. It's really only just a nitpick because this is a self-contained helper and maybe should be done in a separate patch atop these all since even other functions used only here are not marked as static.

Reply:
Actually, most of this can be dismissed becuse only now did I get to the commit where you split the file, so there the functions even can't be static..